plugins/rollbar_dashboards/app/views/rollbar_dashboards/_deploy.html.erb

```diff
@@ -0,0 +1,29 @@
+<%
+  occurrence_term = 'occurrences_count'
+  if deploy.succeeded?
+%>
+  <% if deploy.project.rollbar_dashboards_settings.empty? %>
+    <%= render partial: 'rollbar_dashboards/no_dashboards', locals: { project: deploy.project } %>
+  <%
+    else
+      @data = deploy.project.rollbar_dashboards_settings.map do |dashboard|
+        params = {
+          date_from: deploy.created_at.in_time_zone(dashboard.time_zone).strftime('%m-%d-%Y %H:%M:%S'),
```

Comment:
would prefer if only users have a time zone and dashboards have not

Reply:
... we got time display helpers for that flying around somewhere

Reply:
The time zone of the Rollbar project is independent of the time zone of the user, so we'd want to store that separately in order to get the correct items for the deploy dashboard.